docs/reference/api/nodejs_README.md

```diff
@@ -0,0 +1,42 @@
+---
```

Comment:
why are there 2 node.js API pages?

Reply:
That is a good question :-) I've removed the duplicate.